views/includes/map-script.ejs

```diff
@@ -0,0 +1,63 @@
+<script>
+  const platform = new H.service.Platform({ apikey: 'QeQ1GSfDvpTOIJTJtLhZ5NT2sc2j3a3nPYLw_8XDEu0' });
```

Comment:
Shouldn't this go in .env? @AvishekGhost

Reply:
You can name it HEREMAPS_API_KEY, I've added it in the Heroku build config

Reply:
Okay